src/main/java/uk/gov/hmcts/ccd/domain/service/getcasedocument/CaseDocumentMetadataMapper.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.ccd.domain.service.getcasedocument;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.ReportingPolicy;
+import uk.gov.hmcts.ccd.domain.model.search.CaseDocumentsMetadata;
+
+
+
+@Mapper(unmappedTargetPolicy = ReportingPolicy.IGNORE, componentModel = "spring")
+public interface CaseDocumentMetadataMapper {
+
+    uk.gov.hmcts.reform.ccd.document.am.model.CaseDocumentsMetadata
```

Comment:
Seems like one of the field name is not matching  => "documentHashToken" in CCD model but "documentHashTokens" in the client library.  Correct ccd one to plural.

Reply:
👍  Thought it would be the mapper stuff.  Didn't catch the name disrepancy

Reply:
Fixed in 70dce10